packages/mobile/Gemfile.lock

```diff
@@ -260,6 +260,7 @@ GEM
 PLATFORMS
   ruby
   universal-darwin-19
+  universal-darwin-20
```

Comment:
Is this change needed?

Reply:
This was in the previous release, not sure why it was needed.